features/include/pcl/features/3dsc.h

```diff
@@ -102,17 +104,20 @@ namespace pcl
         min_radius_(0.1),
         point_density_radius_(0.2),
         descriptor_length_ (),
-        rng_alg_ (),
-        rng_ (new boost::uniform_01<boost::mt19937> (rng_alg_))
+        rng_ (),
+        rng_dist_ (0.0f, 1.0f)
```

Comment:
```suggestion
        rng_dist_ (0.0, 1.0)
```

It's a double instantiation.

Reply:
Not anymore. See your own suggested changes below ;)